examples/00-fluent/steady_vortex.py

```diff
@@ -0,0 +1,568 @@
+from ansys.fluent.core.examples import download_file
+from ansys.fluent.core.solver import (  # noqa: E402
+    LIC,
+    CellRegister,
+    CellZoneCondition,
+    Contour,
+    General,
+    Graphics,
+    Initialization,
+    IsoClip,
+    IsoSurface,
+    Materials,
+    Mesh,
+    Methods,
+    Models,
+    NamedExpression,
+    PlaneSurface,
+    RunCalculation,
+    Scene,
+    WallBoundary,
+)
+
+# %%
+# Launch Fluent
+# ^^^^^^^^^^^^^
+#
+# Launch Fluent in 3D double precision solver mode.
+
+solver_session = pyfluent.launch_fluent(
+    mode=FluentMode.SOLVER,
+    dimension=Dimension.THREE,
+    precision=Precision.DOUBLE,
+    cleanup_on_exit=True,
+    cwd=os.getcwd(),
+)
+print(solver_session.get_fluent_version())  # Print the Fluent version
+pyfluent.set_console_logging_level("INFO")  # Set the console logging level
+
+# %%
+# Mesh
+# ^^^^
+#
+# Download the mesh file and read it into the Fluent session.
+
+vortex_mesh = download_file(
+    "vortex-mixingtank.msh.h5",
+    "pyfluent/examples/Steady-Vortex-VOF",
+    save_path=os.getcwd(),
+)
+
+# %%
+# Define Constants
+# ^^^^^^^^^^^^^^^^
+#
+# Define constants.
+
+g = 9.81  # m/s^2
+
+# %%
+#
+# Read Mesh
+# ^^^^^^^^^^^
+#
+# Import the mesh file into the Fluent session.
+
+solver_session.settings.file.read_case(file_name=vortex_mesh)
+
+# %%
+# Display the mesh in Fluent and save the image to a file to examine locally.
+
+# Create a middle plane to display the mesh
+y_mid_plane = PlaneSurface(solver_session, new_instance_name="y_mid_plane")
+y_mid_plane.method = "zx-plane"
+y_mid_plane.y = 0
+y_mid_plane.display()
+
+# Define and display the mesh
+mesh = Mesh(solver_session, new_instance_name="mesh")
+mesh.surfaces_list = y_mid_plane.name()
+mesh.options.edges = True
+mesh.display()
+
+# Create Graphics object to save the mesh image
+graphics = Graphics(solver_session)
+graphics.views.auto_scale()
+if graphics.picture.use_window_resolution.is_active():
+    graphics.picture.use_window_resolution = False
+
+graphics.views.restore_view(view_name="top")
+graphics.views.auto_scale()
+graphics.picture.x_resolution = 600
+graphics.picture.y_resolution = 600
+graphics.picture.color_mode = "mono"
+graphics.picture.save_picture(file_name="mesh.png")
+
+# %%
+# .. figure:: /_static/steady_vortex/mesh.png
+#     :width: 500pt
+#     :align: center
+#
+#     Polyhexcore Mesh for the Steady Vortex Simulation
+
+# %%
+# Define General Settings
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Set gravity
+
+general_settings = General(solver_session)
+general_settings.operating_conditions.gravity.enable = True
+general_settings.operating_conditions.gravity.components = [0.0, 0.0, -g]
+
+# %%
+# Copy Materials from Fluent Database
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Copy water liquid materials from the Fluent database.
+
+copy_material_from_database = Materials(solver_session)
+copy_material_from_database.database.copy_by_name(type="fluid", name="water-liquid")
```

Comment:
@abhishekchitwar 
Shouldn't this just read something like the following?
```suggestion
materials = Materials(solver_session)
materials.database.copy_by_name(type="fluid", name="water-liquid")
```